docs/reference/mpyfiles.rst

```diff
@@ -146,8 +148,8 @@ The .mpy header is:
 size    field
 ======  ================================
 byte    value 0x4d (ASCII 'M')
-byte    .mpy version number
-byte    feature flags
+byte    .mpy major version number
+byte    native arch and minor version number (was feature flags in older versions)
```

Comment:
Is it worth mentioning the details, that the native arch is the top 6 bits, and the minor version is the bottom 2 bits?

Reply:
There is already example Python code showing how to decode the header starting on line 57 in this file. So, I don't think we need to repeat it here.

Reply:
That's actually for decoding `sys.implementation._mpy`.  But, yes, that's the same as the second two bytes of the header.